src/Stratis.Bitcoin.Features.MemoryPool/MempoolBehavior.cs

```diff
@@ -291,20 +296,25 @@ public override object Clone()
             //  filterrate = pto->minFeeFilter;
             //}
 
-            List<TxMempoolInfo> sends = await this.manager.MempoolLock.WriteAsync(() =>
+            List<TxMempoolInfo> sends = new List<TxMempoolInfo>();
```

Comment:
Please add an issue to implement the above code
`if (connman.OutboundTargetReached(false) && !pfrom->fWhitelisted)`

Reply:
var?